DylibBinder/DBGenericParameter.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using Dynamo.SwiftLang;
+using SwiftReflector;
+using SwiftReflector.Inventory;
+
+namespace DylibBinder {
+	public class DBGenericParameter {
+		public DBGenericParameter (int depth, int index)
+		{
+			Depth = depth;
+			Index = index;
+			Name = SLGenericReferenceType.DefaultNamer (depth, index);
+		}
+
+		public int Depth { get; }
+		public int Index { get; }
+		public string Name { get; }
+	}
+
+	public class DBGenericParameters {
+
+		public DBGenericParameters (SwiftBaseFunctionType signature) {
+			AssignGenericParameters (signature);
+		}
+
+		public DBGenericParameters (SwiftType swiftType)
+		{
+			FilterGenericParameterSource (swiftType);
```

Comment:
This one here, does look like a nice candidate for a static method, and therefor a extension, right?